internal/kubeconfig/kubeconfig.go

```diff
@@ -15,7 +15,7 @@ type ReadWriteResetCloser interface {
 }
 
 type Loader interface {
-	Load() (ReadWriteResetCloser, error)
+	Load(string) (ReadWriteResetCloser, error)
```

Comment:
change this to []ReadWriteResetCloser so we can have this return multiple kubeconfigs in the future. then read the [0] at the call-site and add a `// TODO`